aptos-move/framework/aptos-framework/sources/aptos_governance.move

```diff
@@ -127,7 +127,7 @@ module aptos_framework::aptos_governance {
         signer_address: address,
         signer_cap: SignerCapability,
     ) acquires GovernanceResponsbility {
-        system_addresses::assert_aptos_framework(aptos_framework);
+        system_addresses::assert_framework_reserved_address(aptos_framework);
```

Comment:
Why does this need to change? You can still pass the aptos_framework account here right?